pkg/cloud/awsprovider.go

```diff
@@ -1980,7 +1984,7 @@ func parseSpotData(bucket string, prefix string, projectID string, region string
 				continue
 			}
 
-			klog.V(4).Infof("Found spot info %+v", spot)
+			klog.V(1).Infof("Found spot info for: %s", spot.InstanceID)
```

Comment:
Is this log potentially noisy?

Reply:
If it's only refreshing 15 minutes, not sure why it would be?